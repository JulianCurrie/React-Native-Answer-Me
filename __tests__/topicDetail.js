import 'react-native';
import React from 'react';
import TopicDetail from '../src/components/topicDetail';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <TopicDetail />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
