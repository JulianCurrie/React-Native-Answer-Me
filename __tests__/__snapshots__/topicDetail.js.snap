exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "azure",
      "flex": 1,
    }
  }>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginTop": 20,
        "padding": 10,
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "blue",
            "marginTop": 10,
          }
        }>
        Back
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignSelf": "center",
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 20,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      } />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 14,
          "textAlign": "center",
        }
      } />
    <TextInput
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Add your thoughts"
      style={
        Object {
          "alignSelf": "center",
          "borderColor": "black",
          "borderRadius": 5,
          "borderWidth": 1,
          "height": 50,
          "marginTop": 10,
          "textAlign": "center",
          "width": 250,
        }
      } />
    <ScrollView
      dataSource={
        ListViewDataSource {
          "items": 0,
        }
      }
      enableEmptySections={true}
      initialListSize={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={1000}
      onKeyboardDidHide={undefined}
      onKeyboardDidShow={undefined}
      onKeyboardWillHide={undefined}
      onKeyboardWillShow={undefined}
      onLayout={[Function]}
      onScroll={[Function]}
      pageSize={1}
      removeClippedSubviews={true}
      renderRow={[Function]}
      scrollEventThrottle={50}
      scrollRenderAheadDistance={1000}
      stickyHeaderIndices={Array []}
      style={
        Object {
          "flex": 1,
          "margin": 20,
        }
      } />
  </View>
</View>
`;
